src/transaction.ts

```diff
@@ -1292,12 +1336,14 @@ export class Transaction extends Dml {
   constructor(
     session: Session,
     options = {} as spannerClient.spanner.v1.TransactionOptions.ReadWrite,
-    queryOptions?: IQueryOptions
+    queryOptions?: IQueryOptions,
+    requestOptions?: Pick<IRequestOptions, 'transactionTag'>
```

Comment:
In most cases, this constructor is called by the `_prepareTransaction` method in the session pool here: https://github.com/googleapis/nodejs-spanner/blob/d3a63c839a902ccb7de8c0270921202f18b93370/src/session-pool.ts#L1146

That means that the `requestOptions` will be undefined and not used. This means that any request options with a transaction tag that is passed in to one of the [run transaction methods](https://github.com/googleapis/nodejs-spanner/blob/d3a63c839a902ccb7de8c0270921202f18b93370/src/database.ts#L2603) will not be used, and no transaction tag will actually be used for the requests on the transaction.